### PR TITLE
Date picker bug

### PR DESCRIPTION
**Pull request by MrHardcode**
date_picker_bug

### CODE REVIEW

File: src/components/Date.jsx [hunk 1]
```diff
@@ -167,6 +167,13 @@ const MyStartDateSelector = ({
     <>
       <DatePicker
         selected={startDate}
+        /**
+         * This onChange method only runs when the date picker component is renderen (of course)
+         * This means that a user can pick a bad date, re-render the component and confirm the bad date 
+         * since the date check only runs on change (when the user clicks another date) and the date 
+         * the user chose last is saved even if the date picker is re-rendered
+         * To fix this: Reset date every time the date picker is re-rendered
+         */
```
Comment (MrHardcode) [inline src/components/Date.jsx]: Retarded explanation of the problem, but I hope you get the point

Comment (Castau) [reply src/components/Date.jsx]: So it needs a clear/clan-up function? Or am I misunderstanding? But maybe hook the datepicker component up with the map first - there might be more bugs after that.

Comment (MrHardcode) [reply src/components/Date.jsx]: Yes I think saving the initial dates that are shown when the component is rendered first time would help solve the problem - then we can set the date to the initial value every time the component re-renders. Solving the problem this way also means that we don't have to change anything in the onChange-method. 
I don't think we have to wait till the date picker has been hooked up with the map. But there's also no rush and you're right - maybe more bugs will appear which could be included in this bug-fix thread.

**Comment by MrHardcode** [comment]
Should be fixed now

**Review by Castau — APPROVED** [review]

**Comment by Castau** [comment]
> Should be fixed now

nice!

**Review by Runi-VN — APPROVED** [review]

**Pull request merged**